src/aat/resources/features/F-111 - Remove Case-Assigned Users and Roles/With Organisation Context/common/S-111.13/S-111.13_Add_Case_Assigned_User_Roles_for_Case_C1.td.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Must successfully assign a user and case role for a specific case by a user calling through/from an authorised application",
+
+  "_guid_": "S-111.13_Add_Case_Assigned_User_Roles_for_Case_C1_With_Organisation",
+  "_extends_": "F-111_Add_Case_Assigned_User_Roles_Base",
+
+  "specs": [
+    "by Dil, within the context of his organisation, to add a Case Role - CR1 on C1 for Olawale and Hemanth"
+  ],
+
+  "users": {
+    "_extends_": "F-111_Users"
+  },
+
+  "prerequisites": [
+    "F-111_Prerequisite_Counter_Check_Call"
+  ],
+
+  "request": {
+    "headers": {
+      "Authorization": "Bearer ${[scenarioContext][testData][users][userDil][accessToken]}"
+    },
+    "body": {
+      "case_users": [
+        {
```

Comment:
That block there is actually needed here:
https://github.com/hmcts/ccd-data-store-api/pull/1045/files#diff-7471b113100f3aaca0e72f80170d2ee8R23-R26

Reply:
Sorry - this is the request payload - just ignore this comment.